test/unit/component/parser/json/BpmnJsonParser.label.test.ts

```diff
@@ -22,11 +22,11 @@ describe('parse bpmn as json for label font', () => {
     ['exclusiveGateway'],
     ['inclusiveGateway'],
     ['parallelGateway'],
+    ['callActivity'],
```

Comment:
:question: why changing the order (same question as for `BpmnJsonParser.label.bounds.test.ts`)?

Reply:
I added some scpecific tests for call Activity, and after remove them in order to comme back to the previous tests, but I didn't remember the order 😉